GOVERNANCE.md

```diff
@@ -0,0 +1,162 @@
+🚨 This document is not ratified 🚨
+In the process of creating a charter, many "process" and governance related things were included in the draft of the charter. Feedback was that the charter should not include such content, and that it should be moved to a separate file. This is that file.
+The initial commit includes content already approved by the group, however some additional content will be added at a later point, and this comment removed.
+For context, see https://github.com/json-schema-org/community/pull/325
+
+---
+
+# Governance
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## JSON Schema Governing Body (TSC)
```

Comment:
I have been thinking lately about this and I do think that we should add a clause to limit the number of members of the TSC working for the same company to make sure there is balance and equity in all decisions and also to make sure there is a positive external perception of all decision.

AsyncAPI do this:
No more than one-fourth of the TSC members may be affiliated with the same employer. If the removal or resignation of a TSC member, or a change of employment/affiliation by a TSC member, creates a situation where more than one-fourth of the TSC membership shares an employer, then the situation must be assessed by TSC. In any case, the number of TSC members affiliated with the same employer must not be higher than one-third (round half up number).

Nodejs do this:
No more than one-fourth of the TSC voting members may be affiliated with the same employer. If change in TSC voting membership or a change of employment by a TSC voting member creates a situation where more than one-fourth of the TSC voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member status from one or more TSC voting members affiliated with the over-represented employer(s).

Reply:
Previous conversation does agree with doing this, but we find ourselves in the position where the vast majority of the proposed TSC is already employed by the same company simply because they hired members of the core team.  We don't have sufficient membership to satisfy a condition like this right now.

Reply:
> While at this time there are no limits on TSC membership by employer, the TSC will strive to keep to at least less than 50%, ideally 33% (One third, one in three). The TSC will re-evaluate this specific clause at least every six months, and aim to revise it within one year to meet the "no more than 1/3 employer member affiliation" mandate.

I think we should come up with an specific number of TSC members to start applying the percentage of employer representation. For example, I suggest we establish 10 TSC as the number as the size to automatically enable this rule without having to re-evaluate every 6 months. 

Mu suggestion for this paragraph is:

While the project is not looking to obtain "Impact" project status within the OpenJS Foundation, there is no requirement set out to limit the number of TSC members by employer. It is in some cases considered difficult or even unhelpful for the project to limit the number or percentage of TSC members by employer (especially when an employer has employed individuals already active in the community to work exclusively on the open source project). While at this time there are no limits on TSC membership by employer, as soon as the TSC reach 10 members a rule will be enabled to set the limit to at least less than 50%, ideally 33% (One third, one in three).

Reply:
I'm ok with this suggestion.

Reply:
I made this change and removed the specific OpenJS Foundation mention.